src/css/custom.css

```diff
@@ -419,14 +439,53 @@ div.markdown h3 a:hover {
 
 .mediaImage {
   width: 100%;
-  height: auto
+  height: auto;
 }
 
 /* remove extra vertical space from tabs when it contains a code block */
-.tabs-container .theme-code-block {
-  margin-top: -1.4rem;
+.tabs-container .margin-top--md .theme-code-block {
+  margin-top: -1rem;
+}
+
+/* code title styling */
+div.codeBlockTitle_node_modules-\@docusaurus-theme-classic-lib-theme-CodeBlock-Content-styles-module {
+  font-size: 1.05rem;
+  font-weight: 700;
+  color: var(--ifm-color-gray-300);
+  border-bottom: 3px solid var(--ifm-code-background-color-light);
 }
 
 code span::selection {
   background: var(--docusaurus-highlighted-code-line-bg);
 }
+
+.tabs-container {
+  background-color: var(--ifm-code-background-color);
+  border-radius: var(--ifm-code-border-radius);
+}
+
+.tabs-container .tabs {
+  box-shadow: inset 0px -6px 0px -3px var(--ifm-code-background-color-light);
+}
+
+.tabs-container .tabs__item {
+  margin: 0;
+  padding: 0.75rem 1rem;
+  font-size: 1.05rem;
+}
+
+.tabs-container .tabs__item:hover {
+  background-color: var(--ifm-tabs-hover-color);
+}
+
```

Comment:
One other thing I think is a nice touch is hiding the title if there is one with tabs. Feels pretty clunky like this

![Screen Shot 2022-11-03 at 9 41 49 AM](https://user-images.githubusercontent.com/6298501/199752097-cfb4de16-92bc-40d6-bb12-307b8ad58aa5.png)

```suggestion
.tabs-container div[class^='codeBlockTitle'] {
  display: none;
}
```

Reply:
I agree its clunky but I don't want the title to disappear, its important the user knows which file the code block belongs to.

Reply:
Is there an option for the tabs to be the file name like we currently have? Or is it subjective based on the content that something like the tabs in the above image should have that text?

Reply:
Not sure what you mean. Nothings changed here but the style. It looks more prominent now because the background color of the tab container is the same as the code block.

Reply:
Sorry, to clarify, I meant like we currently have with the file name as the tab text on production. Is it deliberate to name the tabs something different and if so does warrant having the tabs - filename - codeblock sandwich? 

![Screen Shot 2022-11-03 at 5 13 51 PM](https://user-images.githubusercontent.com/6298501/199835829-d4cdc143-4276-4335-80a6-08693d542738.png)

Reply:
The reason the filename isn't in the tab in production is because it had to go there as the old docs don't support title attributes on code blocks. I'm not sure what code example this is and not sure if it still exists in the new CT docs. We can probably reorganize how the tabs and titles are done, but I would consider that out of scope for this css update.